api/baseline-api-v1.0.0.yaml

```diff
@@ -0,0 +1,4382 @@
+          enum:
+            - offline_access
+          readOnly: false
+        expires_in:
+          type: number
+          description: Time in minutes until `Token` will no longer be valid
+          readOnly: true
+          nullable: true
+        access_token:
+          type: string
+          description: JSON formatted Web Token
+          format: password
+          readOnly: true
+          nullable: true        
+        refresh_token:
+          type: string
+          description: JSON formatted Web Token
+          format: password
+          readOnly: true
+          nullable: true
+        token:
+          type: string
+          description: JSON formatted Web Token
+          format: password
+          readOnly: true
+          nullable: true
+        permissions:
+          type: number
+          description: Authorized permissions for `Token`, expressed as a bitmask
+          nullable: true 
+          readOnly: true
+        organization_id:
+          type: string
+          description: UUID of `Organization` requiring authorization. _UUID as specified by RFC4122_
+          nullable: true
+          writeOnly: true
+        application_id:
+          type: string
+          description: ID of `Workgroup` requiring authorization. _UUID as specified by RFC4122_
+          nullable: true
+          writeOnly: true
+      example:
+        id: r19e831-43fd-237e-8b43-12af828ba3e3
+        scope: offline_access
+        expires_in: 900
+        access_token: <JWT>
+        refresh_token: <JWT>
+        permissions: 415
+    Transaction:
+      properties:
+        id:
+          type: string
+          description: UUID for a specific `Transaction`. _UUID as specified by RFC4122_
+          format: uuid
+          readOnly: true
+        created_at:
+          type: string
+          description: Date and time of `Transaction` creation
+          format: date-time
+          readOnly: true
+        network_id:
+          type: string
+          description: UUID of the `Network` on which the Transaction will be broadcast. _UUID as specified by RFC4122_
+          format: uuid
+          readOnly: false
+        user_id:
+          type: string
+          description: UUID for the `User` that created `Transaction`. _UUID as specified by RFC4122_
+          format: uuid
+          readOnly: true
+        wallet_id:
+          type: string
+          description: UUID of `Wallet` to be used to sign `Transaction`. _UUID as specified by RFC4122_
+          format: uuid
+          readOnly: false
+        key_id:
+          type: string
+          description: UUID of the Vault Key which will be used to sign the `Transaction`; required in the absence of a `signer`. _UUID as specified by RFC4122_
+          format: uuid
+        account_id:
+          type: string
+          description: UUID of `Account` to be used for `Transaction` creation. _UUID as specified by RFC4122_
+        hd_derivation_path:
```

Comment:
that is an implementation specific name ... wallet/key derivation path is better